creditcoin-js/src/transforms.ts

```diff
@@ -31,11 +36,94 @@ import {
     TransferKind,
     UnverifiedCollectedCoins,
     CollectedCoins,
+    Blockchain,
+    Currency,
+    EvmTransferKind,
 } from './model';
 
+export const createEvmInfo = (info: PalletCreditcoinPlatformEvmInfo): EvmInfo => {
+    const chainId = info.chainId.toString();
+    return {
+        chainId: new BN(chainId),
```

Comment:
Not sure I follow, what do you mean by this?

Reply:
Using the chainId constructor instead of BN

Reply:
since `EvmChainId` is just an alias for `BN` so there is no real way to create it as an `EvmChainId`. You could do `new BN(chainId) as EvmChainId` but it seems overly verbose.

Reply:
> You could do new BN(chainId) as EvmChainId but it seems overly verbose.

That or make `EvmChainId` a class

Reply:
I didn't know that huh, is it the same in most languages? So you can't 'new' a type alias?